src/tests/record.rs

```diff
@@ -284,17 +287,30 @@ async fn record_http(req: Request<Body>, client: Client) -> Result<ResponseAndEx
             general_purpose::STANDARD.encode(&body)
         },
     };
+    warn!(?request);
+
+    let (status, headers, body) = if should_record() {
+        // When we're constructing the outbound request, if the request is to the _actual_ S3,
+        // we're going to replace the HTTP protocol with HTTPS. This is the flip side of the hard
+        // coding of "http" as the protocol in Base::test().
+        //
+        // This is, admittedly, hacky as hell.
+        let uri = match uri.host() {
+            Some(host) if host.ends_with(".amazonaws.com") => {
+                let uri = uri.to_string().replace("http://", "https://");
+                uri.parse::<Uri>().unwrap()
+            }
+            _ => uri,
+        };
 
-    let (status, headers, body) = if let Ok("passthrough") = dotenvy::var("RECORD").as_deref() {
         // Construct an outgoing request
-        let uri = uri.to_string().replace("http://", "https://");
-        let uri = uri.parse::<Uri>().unwrap();
         let mut req = Request::builder()
             .method(method.clone())
             .uri(uri)
             .body(body.into())
             .unwrap();
         *req.headers_mut() = headers.clone();
+        info!(?req);
```

Comment:
was this intentionally left in? (same for the `warn!()` above)

Reply:
…fuck.